internal/commands/list_trusted_builders.go

```diff
@@ -0,0 +1,32 @@
+package commands
+
+import (
+	"github.com/spf13/cobra"
+
+	"github.com/buildpacks/pack/internal/config"
+	"github.com/buildpacks/pack/logging"
+)
+
+func ListTrustedBuilders(logger logging.Logger, cfg config.Config) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "list-trusted-builders",
+		Short: "List Trusted Builders",
+		Long:  "List Trusted Builders.\n\nShow the builders that are either trusted by default or have been explicitly trusted locally using `trust-builder`",
+		RunE: logError(logger, func(cmd *cobra.Command, args []string) error {
+			logger.Info("Trusted Builders:")
+
+			for _, builder := range suggestedBuilders {
+				logger.Infof("  %s", builder.Image)
+			}
+
+			for _, builder := range cfg.TrustedBuilders {
+				logger.Infof("  %s", builder.Name)
+			}
+
+			return nil
+		}),
+	}
+
+	AddHelpFlag(cmd, "trust-builder")
```

Comment:
```suggestion
	AddHelpFlag(cmd, "list-trusted-builders")
```

Reply:
Is this line actually necessary at all? Even with the existing line the output works as expected

```
$  ./out/pack help list-trusted-builders
List Trusted Builders.

Show the builders that are either trusted by default or have been explicitly trusted locally using `trust-builder`

Usage:
  pack list-trusted-builders [flags]
...
```

and it doesn't break `trust-builder` either

```
$  ./out/pack help trust-builder
Trust builder.

When building with this builder, all lifecycle phases will be run in a single container using the builder image.

Usage:
  pack trust-builder <builder-name> [flags]
```

Reply:
Yes, at least historically, IIRC it's purpose is to override the help flag message and capitalizing it to be consistent.

The output you presented omitted the flags section, that's where it would be apparent.

Reply:
Got it, updated